gems/spree/OSVDB-90865.yml

```diff
@@ -0,0 +1,20 @@
+---
+gem: spree
```

Comment:
Also after examining the Source Code it would appear that spree_auth before 1.1.6 (which was never officially released) is also vulnerable. Its my understanding that 1.2.0 and 1.3.0 aren't actually vulnerable. The said file doesn't exist in these versions